onnx_tf/handlers/backend/clip.py

```diff
@@ -2,33 +2,51 @@
 
 from onnx_tf.handlers.backend_handler import BackendHandler
 from onnx_tf.handlers.handler import onnx_op
-from onnx_tf.handlers.handler import tf_func
 
 
 @onnx_op("Clip")
-@tf_func(tf.clip_by_value)
 class Clip(BackendHandler):
 
+  @classmethod
+  def args_check(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    # uint64 cannot upcast to any tensorflow supported datatype
+    # for tf.clip_by_value that didn't lose precision
+    if x.dtype == tf.uint64:
+      exception.OP_UNSUPPORTED_EXCEPT(
```

Comment:
missing declaration for `from onnx_tf.common import exception`

Reply:
@pluradj good catch, I just updated the code. please review it again.